### PR TITLE
Add readme for the workshop

### PR DESCRIPTION
**Pull request by RobbeSneyders**

### CODE REVIEW

File: components/comments_filtering/.output_manifest.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (NielsRogge) [inline components/comments_filtering/.output_manifest.json:1]: Any reason this file is included?

**Review by NielsRogge — COMMENTED** [review]
Some comments:

- what's wrong with keeping all components in the starcoder folder?
- why not a single build_images.sh script?
- shouldn't we make the users create all files for a new component themselves? i.e. Dockerfile, requirements.txt, main.py, component spec

File: pipeline.py [hunk 2]
```diff
@@ -1,31 +1,26 @@
...
 from fondant.logger import configure_logging
+from fondant.pipeline import ComponentOp, Pipeline, Client
+from pipeline_config import PipelineConfig
 
 configure_logging()
 logger = logging.getLogger(__name__)
 
 # Pipeline description
 team_name = "cheesecake"  # TODO: insert your team name
-hf_token = ""  # TODO: insert your HF token
```
Comment (NielsRogge) [inline pipeline.py:18]: If we remove this, then users won't be able to run the first component

File: components/comments_filtering/.output_manifest.json (hunk 1 above)
Comment (RobbeSneyders) [reply components/comments_filtering/.output_manifest.json:1]: To run locally

File: pipeline.py (hunk 2 above)
Comment (RobbeSneyders) [reply pipeline.py:18]: I tested this and it works. It is not used.

**Comment by RobbeSneyders** [comment]
> what's wrong with keeping all components in the starcoder folder?

I don't think it's necessary. This is the structure I would use for a simple repo containing a single pipeline.

> why not a single build_images.sh script?

This was already split before this PR. The users only need to build their own component, so I think it's fine.

> shouldn't we make the users create all files for a new component themselves? i.e. Dockerfile, requirements.txt, main.py, component spec

We don't have a lot of time, so I would provide a boilerplate

**Pull request merged**